src/post/Write/Write.js

```diff
@@ -105,14 +105,33 @@ class Write extends React.Component {
         isUpdating: isUpdating || false,
       });
     }
+
+    if (draftId) {
+      this.draftId = draftId;
+    } else {
+      this.draftId = Date.now().toString(16);
```

Comment:
If 2 people are saving there drafts at same time, won't they have the same draftId?

Reply:
They would, but it would only matter if two people on the same account would open editor exactly at the same time. We could use `uuid` for this, just in case :D